esp/src/eclwatch/DFUQueryWidget.js

```diff
@@ -306,8 +316,8 @@ define([
                 arrayUtil.forEach(this.desprayGrid.store.data, function (item, idx) {
                     var request = domForm.toObject(context.id + "DesprayForm");
                     request.destPath = context.desprayTargetPath.getDropZoneFolder();
-                    if (!context.endsWith(request.destPath, "/")) {
-                        request.destPath += "/";
+                    if (!context.endsWith(request.destPath, request.destPath)) {
+                        request.destPath += request.destPath;
```

Comment:
Do I misunderstand the endsWith() here? You should check whether the request.destPath ends with the path separator or not. If not, append it.